src/types/nitro.ts

```diff
@@ -165,7 +165,7 @@ export interface NitroOptions extends PresetOptions {
   // Routing
   baseURL: string,
   handlers: NitroEventHandler[]
-  routes: { [path: string]: NitroRouteOptions }
+  routes: { [path: string]: NitroRouteRules }
```

Comment:
Do you mind also renaming `routes` (we can have backward compatibility for a while)

Reply:
shall we also have config.nitro.routeRules or keep as config.nitro.routes?

Reply:
I mean runtime config.

Reply:
Sure lets rename it!